### DISCUSSION TITLE
Is Tile3DLayer could be used with OrbitView?

### DISCUSSION THREAD

**Discussion by Wincy94** [visgl/deck.gl#5550] [Q&A]
Hello. I am developing an application with Tile3DLayer. I used defaultView at first, but I found that the control of OrbitView is more suitable for the application lately. And I found the Viewport of OrbitView does not contain some props (latitude and longitude) which Tile3DLayer required in the update method.

Is there any better way to use those features together?

**Comment by sclebal** [comment]
Hi @Wincy94, [OrbitView](https://deck.gl/docs/api-reference/core/orbit-view) cannot be used with geospatial layers. If you want the same effect that you achieve with OrbitView, maybe you can define a custom [controller](https://deck.gl/docs/api-reference/core/controller) for the [MapView](https://deck.gl/docs/api-reference/core/map-view) that imitates how OrbitController works.

**Reply by Wincy94** [reply]
Thanks for your reply. I will try this suggestion.
